definitions/TGSA.def

```diff
@@ -0,0 +1,115 @@
+%Bootstrap: docker
+%From: pytorch/pytorch:1.9.0-cuda10.2-cudnn7-runtime
+Bootstrap: docker
+From: pytorch/pytorch:1.12.0-cuda11.3-cudnn8-runtime
+
+%labels
+        MAINTAINER      JJ
+%setup
+    # will be installed and removed in post step
+    cp ./src/Singularity_gpu_fix.sh $SINGULARITY_ROOTFS/
+
+%environment
+        export PATH=/usr/local/TGSA:/usr/local/cuda/bin:$PATH
+        export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
+        export CUDA_INSTALL_DIR=/usr/local/cuda
+        export CUDA_HOME=/usr/local/cuda
+        export MODEL_DIR=/usr/local/TGSA_model
+        echo "export MODEL_DIR=/usr/local/TGSA_model" >> ~/.bashrc
```

Comment:
Why do you need to add this to bashrc

Reply:
previously, I had a CUDA driver issue. These could be removed.